### PR TITLE
Add basic support for i18n

### PR DESCRIPTION
**Pull request by Johngtka**
to analize

### CODE REVIEW

File: src/app/menu/menu.component.ts [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+  template: `
+    <menu>
+      <li>About us</li>
+      <li>Contact</li>
+    </menu>
+  `
+})
+export class MenuComponent {
+
+}
```
Comment (macrog) [inline src/app/menu/menu.component.ts:13]: what is this? we do not need this! please remove all `menu.component.ts` file

File: src/assets/en.json [hunk 2]
```diff
@@ -0,0 +1,4 @@
+{
+  "TITLE": "Biomagnethic-App",
+  "SELECT": "Change language"
```
Comment (macrog) [inline src/assets/en.json:3]: "SELECT" ? it should be just title, nothing else

File: src/assets/es.json [hunk 3]
```diff
@@ -0,0 +1,4 @@
+{
+  "TITLE": "Biomagnética-App",
+  "SELECT": "Cambiar el idioma"
```
Comment (macrog) [inline src/assets/es.json:3]: "SELECT" ? it should be just title, nothing else

File: src/assets/pl.json [hunk 4]
```diff
@@ -0,0 +1,4 @@
+{
+  "TITLE": "Biomagnetyczne-App",
+  "SELECT":"Zmień język"
```
Comment (macrog) [inline src/assets/pl.json:3]: "SELECT" ? it should be just title, nothing else

**Comment by Johngtka** [comment]
select is a info where i can change language witch must be translating.
unless i remove this info to html template

File: src/app/app.component.html [hunk 5]
```diff
@@ -2,13 +2,17 @@
...
 
 <body>
-
+  <app-menu></app-menu>
+  <label>Change language
+    <select #langSelect (change)="translate.use(langSelect.value)">
+      <option *ngFor="let lang of translate.getLangs()" [value]="lang" [selected]="lang === translate.currentLang">{{
+        lang }}</option>
+    </select>
+  </label>
```
Comment (macrog) [inline src/app/app.component.html:15]: not needed, not part of this task, there should be just title, nothing else

File: src/app/app.component.spec.ts [hunk 6]
```diff
@@ -1,12 +1,32 @@
...
+        HttpClientTestingModule,
+        TranslateModule.forRoot({
+          loader: {
+            provide: TranslateLoader,
+            useFactory: HttpLoaderFactory,
+            deps: [HttpClient]
+          }
+        })
+      ],
+      providers: [TranslateService]
```
Comment (macrog) [inline src/app/app.component.spec.ts:29]: revert changes please, do not make any changes in spec files for now, there is no point, we will do it everything at the end

File: src/app/app.component.ts [hunk 7]
```diff
@@ -1,9 +1,17 @@
...
+  styleUrls: ['./app.component.css']
 })
 export class AppComponent {
-  title = 'Biomagnethic-App';
+  constructor(public translate: TranslateService) {
+    translate.addLangs(['en', 'pl', 'es']);
+    translate.setDefaultLang('en');
+
+    // const browserLang = translate.getBrowserLang();
+    // translate.use(browserLang.match(/en|pl/));
```
Comment (macrog) [inline src/app/app.component.ts:15]: remove commented out code

File: src/app/app.module.ts [hunk 8]
```diff
@@ -1,16 +1,35 @@
-import { NgModule } from '@angular/core';
 import { BrowserModule } from '@angular/platform-browser';
-
+import { NgModule } from '@angular/core';
 import { AppComponent } from './app.component';
+import { HttpClient, HttpClientModule } from '@angular/common/http';
+import { TranslateModule, TranslateLoader } from '@ngx-translate/core';
+import { TranslateHttpLoader } from '@ngx-translate/http-loader';
+import { MenuComponent } from './menu/menu.component';
```
Comment (macrog) [inline src/app/app.module.ts:7]: remove `MenuComponent`, not needed not a part of this task

File: src/app/app.module.ts [hunk 9]
```diff
@@ -1,16 +1,35 @@
...
-    AppComponent
+    AppComponent,
+    MenuComponent
   ],
   imports: [
-    BrowserModule
+    BrowserModule,
+    HttpClientModule,
+    TranslateModule.forRoot({
+      // defaultLanguage: 'en',
```
Comment (macrog) [inline src/app/app.module.ts:22]: remove commented out code

File: src/assets/en.json [hunk 10]
```diff
@@ -0,0 +1,3 @@
+{
+  "TITLE": "Biomagnethic-App"
```
Comment (macrog) [inline src/assets/en.json:2]: please keep it all in this format:


  ```
    {
        "APP": {
              "TITLE": "Bio magnes"
          }
      }
```

File: src/assets/es.json [hunk 11]
```diff
@@ -0,0 +1,3 @@
+{
+  "TITLE": "Biomagnética-App"
+}
```
Comment (macrog) [inline src/assets/es.json:3]: all `*.json` files should be under i18n folder

**Pull request closed without merging**